content/_assets/styles/components/q-figure.scss

```diff
@@ -386,39 +395,39 @@
       }
     }
 
-    .quire-grid--2 {
-      width: 49%;
+    .quire-grid--2 > * {
+      width: 50%;
```

Comment:
@geealbers I had to change where we were applying the `.quire-grid--#` classes, and the starter only has an instance of a **`2`** column grid. These percentages seem to be set intentionally slightly lower than the intended amount (49% vs 50% here). I had to adjust it back to 50% to match.

I think the DOM changes in https://github.com/thegetty/quire/pull/893 might have unintentionally fixed a thing that was requiring these weird percentages, but I am going to hold off on adjusting the other column counts until we have testbed examples of `figureGroup`s with all possible column counts